benchmarking/tpch/__main__.py

```diff
@@ -126,6 +126,7 @@ def run_all_benchmarks(
     csv_output_location: str | None,
     ray_job_dashboard_url: str | None = None,
     requirements: str | None = None,
+    no_pymodules: bool = False,
```

Comment:
```suggestion
    pymodules: bool = True,
```

Prefer single positive over double negative

Reply:
Updated.

The reason why it was a double-negative in the first place is because I wanted the "omission" of a flag to represent `False` and the inclusion of it to represent `True`, and I wanted the default behaviour to select `True`. However, that leads to weirdness with sending in flags through the CLI.

I.e., `python -m benchmarking.tpch --flag` would result in `flag=True`, but `python -m benchmarking.tpch` would result in `flag=True` as well (given that the exclusion of `flag` results in `True` being used). Therefore, can't really specify `flag=False` anymore.

Updated the invocation so that you have to specify a boolean value. I.e., `python -m benchmarking.tpch --flag=False`.